File: app/src/main/java/co/tpcreative/supersafe/common/request/ChangeUserIdRequest.java
package co.tpcreative.supersafe.common.request;

import java.io.Serializable;

import co.tpcreative.supersafe.common.util.Utils;

public class ChangeUserIdRequest implements Serializable {
   public String user_id;
   public String new_user_id;
   public String  _id;
   public String session_token;
   public ChangeUserIdRequest(VerifyCodeRequest request){
       this.user_id = request.user_id;
       this.new_user_id = request.new_user_id;
       this._id = request._id;
       this.session_token = Utils.getAccessToken();
   }
}

File: app/src/main/java/co/tpcreative/supersafe/common/request/ChangeUserIdRequest.kt
package co.tpcreative.supersafe.common.request
import co.tpcreative.supersafe.common.util.Utils
import java.io.Serializable

class ChangeUserIdRequest : Serializable {
    var user_id: String?
    var new_user_id: String?
    var _id: String?
    var session_token: String?

    constructor(request : VerifyCodeRequest){
        this.user_id = request.user_id
        this.new_user_id = request.new_user_id
        this._id = request._id
        this.session_token = Utils.getAccessToken()
    }

    /*Changed email*/
    constructor(newUser : String){
        val mUser = Utils.getUserInfo()
        this.user_id = mUser?.email
        this.new_user_id = newUser
        this._id = mUser?._id
        this.session_token = Utils.getAccessToken()
    }
}